### PR TITLE
Sso anbindung in Stack

### PR DESCRIPTION
**Pull request by MrSebastian**
**Description**

- adding keycloak to docker-stack
  - with migration container for config
  - with postgresql-db
- changing profiles for backend and frontend to use security
- updating documentation
- the possibility to connect the local service, that is in development is done separately

### CODE REVIEW

File: stack/README.md [hunk 1]
```diff
@@ -15,4 +19,16 @@ docker compose up -d
 docker compose --profile frontend up -d
 ```
 
-additional configuration for the services is done via `*.env`-Files
\ No newline at end of file
+additional configuration for the services is done via `*.env`-Files
+
+### Keycloak
+
+An admin user and a test user ist created by `init-keycloak`. You can change the configuration via the keycloak ui.
```
Comment (LuisEinig) [inline stack/README.md:26]: *ist* should probably be *are* here

File: stack/README.md [hunk 2]
```diff
@@ -15,4 +19,16 @@ docker compose up -d
...
+
+#### configuration migration
+
+Realm, client user and other configuration should be done by the migration client. Its config files are located in
+`keycloak\migartion`. The main file ist `keycloak-changelog.yml`. It contains the list of migration files that
```
Comment (LuisEinig) [inline stack/README.md:31]: same here

File: stack/README.md (hunk 1 above)
Comment (MrSebastian) [reply stack/README.md]: resolved with 3326e69820529c51b5056c4a83914b9959155115

File: stack/README.md (hunk 2 above)
Comment (MrSebastian) [reply stack/README.md]: resolved with 3326e69820529c51b5056c4a83914b9959155115

**Review by LuisEinig — APPROVED** [review]

**Pull request merged**